include/fty_common_db_asset.h

```diff
@@ -77,6 +77,10 @@ int select_asset_element_all_with_warranty_end(tntdb::Connection& conn, std::fun
 // returns 0 if succesful
 // returns -1 if error occurs
 
+int select_assets_by_container(tntdb::Connection& conn, uint32_t element_id, std::vector<uint16_t> types,
```

Comment:
add a comment explaining that api with configured arg.

Reply:
Take into account